networking/tunnel/geneve/basic/runtest.sh

```diff
@@ -235,8 +236,10 @@ else
 			rlRun "tcpdump -r grev4.pcap -nnle | grep \"$SER_ADDR6.*> $CLI_ADDR6.*Geneve.*vni 0x4d2, proto TEB (0x6558).*$s_geneve1_mac > $c_geneve1_mac.*$gre_s_ip4 > $gre_c_ip4\""
 			[ $? -ne 0 ] && rlRun -l "tcpdump -r grev4.pcap -nnle"
 		fi
-		rlRun "$C_CMD tcpdump -i any -w grev6.pcap &"
+		rlRun "$C_CMD tcpdump -U -i any -w grev6.pcap &"
+		rlRun "sleep 5"
 		rlRun "$C_CMD ping6 $gre_s_ip6 -c 5"
+		rlRun "sleep 5"
```

Comment:
Yikes, that's lots of `sleep` calls :sob: Also, isn't 5 seconds extremely high?

Reply:
on part systems like S390x, it would take long before tcpdump get ready, although I don't know why.